webdriver/tests/support/http_handlers/geolocation_override.py

```diff
@@ -0,0 +1,16 @@
+import json
+
+
+def main(request, response):
+    response.headers.set(b"Content-Type", b"application/json")
+    response.headers.set(b"Cache-Control", b"no-cache")
+    response.content = json.dumps(
+        {
+            "status": "OK",
+            "location": {
+                "lat": 37.41857,
+                "lng": -122.08769,
```

Comment:
It would be nice to have a mock server that can be configured by the tests, but that doesn't seem to be easy or not possible without the proper APIs. So that's a good start.

Should we maybe mock all the entries have a specific value?

Reply:
Nope, none of the actual services we use in mozilla-central return other properties, so if we set them here they will not be returned, see: https://searchfox.org/mozilla-central/source/dom/system/NetworkGeolocationProvider.sys.mjs#217-223.

Reply:
Maybe something for later then. Lets make sure that this doesn't break assumptions for other browsers. At least Chrome seems to work based on the test results.